tests/scripts/run-corefx-tests.py

```diff
@@ -261,16 +285,40 @@ def main(args):
     config_args = '-Release -os:%s -buildArch:%s' % (clr_os, arch)
 
     # Run the primary (non-test) corefx build
+    # We used to pass the argument:
+    #    -- /p:CoreCLROverridePath=<path-to-core_root>
+    # which causes the corefx build to overwrite its built core_root with the binaries from
+    # the coreclr build. However, this often causes build failures which breaking changes are
+    # in progress (e.g., a breaking change is made in coreclr that has not yet had compensating
+    # changes made in the corefx repo). Instead, build corefx normally. This should always work
+    # since corefx is protected by a CI testing system. Then, overwrite the built corex
+    # runtime with the runtime built in the coreclr build. The result will be that perhaps
+    # some, hopefully few, corefx tests will fail, but the builds will never fail.
 
     command = ' '.join(('build.cmd' if Is_windows else './build.sh',
-                        config_args,
-                        '-- /p:CoreCLROverridePath=%s' % core_root))
+                        config_args))
 
     log(command)
     returncode = 0 if testing else os.system(command)
     if returncode != 0:
         sys.exit(1)
 
+    # Override the built corefx runtime (which it picked up by copying from packages determined
+    # by its dependencies.props file). Note that we always build Release corefx.
+    # TODO: it might be cleaner to encapsulate the knowledge of how to do this in the
+    # corefx msbuild files somewhere.
```

Comment:
:+1: Could you open an issue in CoreFx explaining our scenario and see if you can get someone to add something like CoreCLROverridePath that only uses the overriding runtime for running, not building?

Reply:
https://github.com/dotnet/corefx/issues/26944